mne/channels/channels.py

```diff
@@ -439,13 +439,13 @@ class UpdateChannelsMixin(object):
     def pick_types(self, meg=True, eeg=False, stim=False, eog=False,
                    ecg=False, emg=False, ref_meg='auto', misc=False,
                    resp=False, chpi=False, exci=False, ias=False, syst=False,
-                   seeg=False, bio=False, ecog=False, include=[],
-                   exclude='bads', selection=None):
+                   seeg=False, dipole=False, gof=False, bio=False, ecog=False,
+                   include=[], exclude='bads', selection=None):
```

Comment:
Strictly speaking it's possible, but realistically this is so far down the parameter list I think the risk of breakage is low enough to be okay (presumably/hopefully people just use kwargs here). We do things like this with plotting functions from time to time.